scripts/fuzz_opt.py

```diff
@@ -414,7 +414,7 @@ def can_run_on_feature_opts(self, feature_opts):
 class FuzzExec(TestCaseHandler):
     def get_commands(self, wasm, opts, random_seed):
         return [
-            '%(MAX_INTERPRETER_ENV_VAR)s=%(MAX_INTERPRETER_DEPTH)d %(wasm_opt)s --fuzz-exec --fuzz-binary %(opts)s %(wasm)s' % {
+            '%(MAX_INTERPRETER_ENV_VAR)s=%(MAX_INTERPRETER_DEPTH)d %(wasm_opt)s --fuzz-exec %(opts)s %(wasm)s' % {
```

Comment:
I think we should get more `--roundtrip` invocations happening before we remove `--fuzz-binary`. right now `--roundtrip` is one one pass among many so it's going to be quite rare to appear at all in the list.

how about in the pick-passes code, with probability 0.5 we put a `--roundtrip` somewhere in the list of passes?

Reply:
I'm going through all this because `--roundtrip` is about to become the only way to transition from a stack IR module back to a normal IR module so the fuzzer will have to add one on all of those transitions as well.